README.md

```diff
@@ -113,12 +115,34 @@ catkin do the following:
       ## TeXlipse
       .texlipse
       ```
+
+#### Multiple Packages
+The _eclipsify_ command can be used in conbitation with the catkin command to easily generate project files for all the ROS packages that  
+reside in a workspace. The options are as follows:
+
+  1. Create project files in the catkin worspace devel directory:
+    ```
+    catkin list -u | xargs -I pkg bash -c "eclipsify pkg"
+    ```
+    As in the single package example, the eclipse project files will be stored in a subdirectory in the **\<catkin workspace>/devel/share/..**  
+    directory.
+
+  2. Create project files in a separate __projects__ directory:
+    Run the following command from the root directory of your catkin workspace
+    ```
+    catkin list -u | xargs -I pkg bash -c "eclipsify pkg -O projects/pkg"
+    ```
+    This will create the eclipse project files for each package in separate subdirectories in the **\<catkin workspace>/projects/** directory.
+
```

Comment:
Where did the in-source option go?

Reply:
I don't think I follow ...

Reply:
For the single package there was the alternative method now introduced with 
> iii In-source option (required by EGit):

It is missing for the multi packages example. 
However, this is obsolete if you like my deduplication suggestion: only explain / mention the `catkin list -u | xargs ...` trick. After my simplification it is the same for all alternatives with respect to where the projects go.